packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -671,7 +669,40 @@
           <Hds::Dropdown::ListItem
             @route="components.dropdown"
             @icon={{if (eq color "critical") "trash" "settings"}}
-            @text="{{state}} with a long multi-line text"
+            @text="{{state}} with a longer text string that may wrap since max-width is defined on the container"
+            @color={{color}}
+            @state={{state}}
+          />
+        {{/each}}
+      </ul>
+    </div>
+  {{/each}}
+
+  <h4 class="dummy-h4">Buttons</h4>
+
+  {{#each @model.ITEM_COLORS as |color|}}
```

Comment:
probably we can merge the two blocks in a single one, with another loop outside. if you want I can try in a separate PR